.eslintrc.js

```diff
@@ -4,17 +4,12 @@ module.exports = {
     node: true,
   },
   parser: '@typescript-eslint/parser',
-  plugins: ['import', 'jsx-a11y', 'prettier'],
-  extends: [
-    'airbnb-typescript',
-    'eslint:recommended',
-    'plugin:@typescript-eslint/recommended',
-  ],
+  plugins: ['import', 'react', 'jsx-a11y', 'prettier'],
+  extends: ['airbnb-typescript', 'eslint:recommended', 'plugin:@typescript-eslint/recommended'],
```

Comment:
Looks like we are moving this back and forth between one line and multiple lines